PagingMenuController.podspec

```diff
@@ -9,19 +9,15 @@
 
 Pod::Spec.new do |s|
   s.name             = "PagingMenuController"
-  s.version          = "0.9.1"
+  s.version          = "0.9.2"
   s.summary          = "A paging view with customizable menu"
-  s.homepage         = "https://github.com/kitasuke/PagingMenuController"
-  # s.screenshots     = "www.example.com/screenshots_1", "www.example.com/screenshots_2"
+  s.homepage         = "https://github.com/chien/PagingMenuController"
```

Comment:
My bad. Will change it back

On Mon, May 16, 2016 at 00:44, Yusuke Kita <notifications@github.com> wrote:
In PagingMenuController.podspec [https://github.com/kitasuke/PagingMenuController/pull/207#discussion_r63309487] :

>    s.summary          = "A paging view with customizable menu"
> -  s.homepage         = "https://github.com/kitasuke/PagingMenuController"
> -  # s.screenshots     = "www.example.com/screenshots_1", "www.example.com/screenshots_2"
> +  s.homepage         = "https://github.com/chien/PagingMenuController"


Why do we have to use your repository?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub [https://github.com/kitasuke/PagingMenuController/pull/207/files/ce9fb443ed31ea355d421b40bf63ee7ad1726385#r63309487]